pkg/model/const.go

```diff
@@ -34,6 +34,7 @@ const (
 	// DefaultSilencePrefix is default etcd prefix for Silences
 	DefaultSilencePrefix             = "/silences"
 	DefaultSilenceValidationInterval = "30s"
+	DefaultSyncSilencesInterval      = "60s"
```

Comment:
Would love to have that documented :D

Reply:
So that users can be aware that their silences might take up to 60 seconds to be syned.